handlers/http.go

```diff
@@ -35,8 +39,9 @@ func stringifyRequest(req common.Request) *bytes.Buffer {
 }
 
 type httpRateLimiter struct {
-	rateLimiter ratelimiter.RateLimiter
-	proxy       http.Handler
+	rateLimiter  ratelimiter.RateLimiter
+	proxy        http.Handler
+	AllowOnError bool // Do not limit on errors when true
```

Comment:
What do you mean that you need to inject the flag? If you mean that you need to change it for test cases, if it's private, the tests in that same package can still get and set it.